PublicAPI.md

```diff
@@ -219,6 +219,125 @@ interface ISettingsService {
 tns.settingsService.setSettings({ userAgentName: "myUserAgent" });
 ```
 
+## debugService
+Provides methods for debugging applications on devices. The service is also event emitter, that raises the following events:
+* `connectionError` event - this event is raised when the debug operation cannot start on iOS device. The causes can be:
+  * Application is not running on the specified iOS Device.
+  * Application is not built in debug configuration on the specified iOS device.
+  The event is raised with the following information:
+```TypeScript
+{
+	/**
+	 * Device identifier on which the debug process cannot start.
+	 */
+	deviceId: string;
+
+	/**
+	 * The error message.
+	 */
+	message: string;
+
+	/**
+	 * Code of the error.
+	 */
+	code: number
+}
+```
+
+* Usage:
+```JavaScript
+tns.debugService.on("connectionError", errorData => {
+	console.log(`Unable to start debug operation on device ${errorData.deviceId}. Error is: ${errorData.message}.`);
+});
+```
+
+### debug
+The `debug` method allows starting a debug operation for specified application on a specific device. The method returns a Promise, which is resolved with a url. The url should be opened in Chrome DevTools in order to debug the application.
+
+The returned Promise will be rejected in case any error occurs. It will also be rejected in case:
+* Specified deviceIdentifier is not found in current list of attached devices.
+* The device, specified as deviceIdentifier is connected but not trusted.
+* The specified application is not installed on the device.
+* Trying to debug applications on connected iOS device on Linux.
+* In case the application is not running on Android device.
```

Comment:
This is valid for iOS devices too. The only difference for now is that if the device is iOS the method will fail after 9 seconds not immediately.